src/components/dialogs/refund.vue

```diff
@@ -1,51 +1,91 @@
-        <v-btn
-          class="px-6 button-blue"
-          id="keep-nr-btn"
-          @click="hideModal()">Keep this Name Request</v-btn>
-      </v-card-actions>
-
-    </v-card>
-  </v-dialog>
+  <span>
+    <v-dialog width="40rem" :value="isConfirmModalVisible" persistent v-if="payments">
+      <v-card>
+
+        <v-card-title class="d-flex justify-space-between">
+          <div>Cancel and Refund</div>
+          <v-btn icon large class="dialog-close" @click="hideConfirmModal()">
+            <v-icon>mdi-close</v-icon>
+          </v-btn>
+        </v-card-title>
+
+        <v-card-text class="text-body-1">
+          This will cancel your Name Request and refund the Name Request fee, Priority Request
+          fee (if any), and Service fees.
+        </v-card-text>
+
+        <v-card-text>
+          <v-alert v-if="fetchError" color="error" icon="mdi-alert" outlined class="my-0" v-html="fetchError" />
+          <refund-summary v-else :payments="payments" />
+        </v-card-text>
+
+        <v-card-text class="text-body-2">
+          The refund will be applied to your original payment method and the requested name will not be
+          examined for use. <template v-if="emailAddress">An email confirming the cancellation and
+          refund of this Name Request will be sent to <strong>{{emailAddress}}</strong>.</template>
+        </v-card-text>
+
+        <v-card-actions class="justify-center">
+          <v-btn
+            class="px-6 button-normal"
+            id="cancel-nr-btn"
+            :loading="loading"
+            @click="confirmRefund()">Cancel this Name Request</v-btn>
+          <v-btn
+            class="px-6 button-blue"
+            id="keep-nr-btn"
+            @click="hideConfirmModal()">Keep this Name Request</v-btn>
+        </v-card-actions>
+
+      </v-card>
+    </v-dialog>
+    <v-dialog width="40rem" :value="isResponseModalVisible" persistent>
+      <v-card>
+
+        <v-card-title class="d-flex justify-space-between">
+          <div>Cancel and Refund</div>
+          <v-btn icon large class="dialog-close" @click="closeResponseModal()">
+            <v-icon>mdi-close</v-icon>
+          </v-btn>
+        </v-card-title>
+
+        <v-card-text class="text-body-1">
+          <span v-html="refundParams.refundMessageText1"></span>
+        </v-card-text>
+
+        <v-card-text
+          v-if="refundParams.refundMessageText2"
+          class="text-body-1">
+          <span v-html="refundParams.refundMessageText2"></span>
+        </v-card-text>
+
+        <v-card-text
+          v-if="refundParams.showStaffContact"
+          class="text-body-2">
+          <contact-info
+            id="tooltip-contact-info"
+            direction="col" />
+        </v-card-text>
+
+        <v-card-actions class="justify-center">
+          <v-btn
+            class="px-6 button-normal"
+            id="keep-nr-btn"
+            @click="closeResponseModal()">OK</v-btn>
```

Comment:
Why is this one "close" but the one at the top is "hide"?

Reply:
I can use the verb 'hide' here.

Reply:
Can you use close for the other one, just to be consistent?

Reply:
👍

Reply:
I misunderstood your previous comment.

Use either verb you want (as long as they're consistent) :)